#----------------------
# Player
#----------------------
int32 PLAYER_1=0
int32 PLAYER_2=1
int32 PLAYER_UNDEFINED=2

#----------------------
# Activity
#----------------------
int32 MOVE=3
int32 WIN=4
int32 DRAW=5
int32 INVALID=6

#----------------------
# Change info
#----------------------
int32 player
int32 activity
